### PR TITLE
Beta7 fixes

### PR DESCRIPTION
**Pull request by tushargupta51**
Fixes #184 #137 #206  #174
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207%23issuecomment-127721811%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207%23issuecomment-128125047%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207%23discussion_r36345000%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/100b790566fcccf53d8bc5fada2ae16e69ae9dc1%23commitcomment-12475874%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207%23issuecomment-127721811%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40brentschmaltz%20%22%2C%20%22created_at%22%3A%20%222015-08-04T19%3A05%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%2C%20%7B%22body%22%3A%20%22make%20the%20comment%20change%20and%20the%20%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-08-05T19%3A40%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3172421%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/brentschmaltz%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Commit%20100b790566fcccf53d8bc5fada2ae16e69ae9dc1%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/100b790566fcccf53d8bc5fada2ae16e69ae9dc1%23commitcomment-12475874%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%23137%20%22%2C%20%22created_at%22%3A%20%222015-07-31T22%3A02%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%5D%2C%20%22title%22%3A%20%22Commit%20100b790%20comments%22%7D%2C%20%22Pull%20c25bcdcb1887626b973f3aee5d1ba05a2ee91b45%20src/System.IdentityModel.Tokens/TokenValidationParameters.cs%2019%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207%23discussion_r36345000%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Just%20add%20%27Gets%20or%20sets%27%20delegate%20for%20validating%22%2C%20%22created_at%22%3A%20%222015-08-05T19%3A40%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3172421%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/brentschmaltz%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/System.IdentityModel.Tokens/TokenValidationParameters.cs%3AL289-298%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/brentschmaltz%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3172421%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/brentschmaltz'></a>

- [ ] <a href='#crh-comment-Commit 100b790566fcccf53d8bc5fada2ae16e69ae9dc1'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/100b790566fcccf53d8bc5fada2ae16e69ae9dc1#commitcomment-12475874'>Commit 100b790 comments</a></b>
- <a href='https://github.com/tushargupta51'></a> #137
- [ ] <a href='#crh-comment-Pull c25bcdcb1887626b973f3aee5d1ba05a2ee91b45 src/System.IdentityModel.Tokens/TokenValidationParameters.cs 19'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207#discussion_r36345000'>File: src/System.IdentityModel.Tokens/TokenValidationParameters.cs:L289-298</a></b>
- <a href='https://github.com/brentschmaltz'></a> Just add 'Gets or sets' delegate for validating
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207#issuecomment-127721811'>General Comment</a></b>
- <a href='https://github.com/tushargupta51'></a> @brentschmaltz
- <a href='https://github.com/brentschmaltz'></a> make the comment change and the :shipit:

<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207?approve=0'></a>&nbsp;<a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by brandwe** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/207'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by tushargupta51** [comment]
@brentschmaltz

File: src/System.IdentityModel.Tokens/TokenValidationParameters.cs [hunk 1]
```diff
@@ -282,9 +289,9 @@ public virtual ClaimsIdentity CreateClaimsIdentity(SecurityToken securityToken,
         }
 
         /// <summary>
-        /// Gets or sets the <see cref="SecurityKey"/> that is to be used for validating signed tokens. 
+        /// Delegate for validating the <see cref="SecurityKey"/> that is to be used for validating signed tokens.
```
Comment (brentschmaltz) [inline src/System.IdentityModel.Tokens/TokenValidationParameters.cs]: Just add 'Gets or sets' delegate for validating

**Comment by brentschmaltz** [comment]
make the comment change and the :shipit:

**Pull request merged**